### DISCUSSION TITLE
ldap Group synchronization

### DISCUSSION THREAD

**Discussion by usercat123** [nilsteampassnet/TeamPass#4189] [General]
I got the user authentification to work easily, but the group synchronization is bugged for me.

so i checked our ldap and these were all the objectClasses:

"objectClass: univentionGroup
objectClass: top
objectClass: sambaGroupMapping
objectClass: posixGroup
objectClass: univentionObject"

i tried every single one in the: "AD Group ObjectClasses attribute", when i check the groups i just see 1 group, but we have more than that.

anyone got a fix?

**Comment by vura** [comment]
![Sem título](https://github.com/user-attachments/assets/8a8f8d1c-3a3d-4f1f-9917-f1fae5f7610e)

I managed to make it work this way.

**Comment by WoutAndAbout** [comment]
@usercat123 did you find what you did wrong?

**Comment by MrLogEN** [comment]
I have a similar problem. I have very simple ldap setup. Let's say that I have 3 groups (objectClass: posixGroup) but when I try to synchronize the roles with the groups, I get only the one that was created last.
I'm kinda stuck at the moment. Anybody knows what might be the cause?

**Comment by adrianowengrat** [comment]
@MrLogEN, i'm stuck on the same problem. Anyone make this work?

**Reply by MrLogEN** [reply]
This fixed it for me https://github.com/nilsteampassnet/TeamPass/issues/4279#issuecomment-2883436053
I was going through issues for some time ...
